jetty-core/jetty-client/src/test/java/org/eclipse/jetty/client/http/HttpReceiverOverHTTPTest.java

```diff
@@ -206,7 +206,7 @@ public void testReceiveBadResponse(HttpCompliance compliance) throws Exception
 
         ExecutionException e = assertThrows(ExecutionException.class, () -> listener.get(5, TimeUnit.SECONDS));
         assertThat(e.getCause(), instanceOf(HttpResponseException.class));
-        assertThat(e.getCause().getCause(), instanceOf(BadMessageException.class));
+        assertThat(e.getCause().getCause(), instanceOf(BadMessage.RuntimeException.class));
```

Comment:
Can't this just be `RuntimeBadMessageException` ?

Reply:
It could be just `BadMessage.class`, but it is a test, so no harm testing for a specific type.